forge/How to make a new extension 3 3/how-to-make-a-new-extension-3-3.md

```diff
@@ -0,0 +1,295 @@
+<!--
+parent: 'TAO 3 1'
+created_at: '2015-06-18 13:17:42'
+updated_at: '2018-10-03 11:59:13'
+authors:
+    - 'Christophe Garcia'
+    - 'Martin Nicholson'
+tags:
+    - 'Legacy Versions:TAO 3.0'
+-->
+
+
+# How to make a new extension in TAO 3.3
+========================================
+
+## Preparation
+--------------
+1.  Install TAO (3.x) (https://github.com/oat-sa/package-tao)
+2.  Download `taoDevTools` extension using composer:
+    `composer require "oat-sa/extension-tao-devtools" —dev`
+3.  Make sure the extension has been installed (See https://hub.taocloud.org/articles/wiki/installing-a-new-extension)
+
+
+## Create the extension
+-----------------------
+1.  Log into TAO BackOffice with your admin account.
+2.  Go to "Settings" (upper right corner) > "Extensions Manager".
+3.  Click on "create new" to create your new extension.
+4.  Fill out the form and call your new extension `taoDocs`.
+    (If you wish, you can select some Samples to generate extra structure).
```

Comment:
Can you find a more generic name? I would probaly go for something like _taoSomeMeaningfulName_